samples/net/cloud/aws_iot_mqtt/Kconfig

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (c) 2023 Lucas Dietrich <ld.adecy@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menu "AWS"
+mainmenu "AWS IoT Core MQTT sample application"
+
+config AWS_ENDPOINT
+	string "AWS endpoint"
+	default ""
+	help
+		Endpoint (hostname) of the AWS MQTT broker.
+		Note that the endpoint is different when using AWS Device Advisor.
+
+config AWS_THING_NAME
+	string "AWS Thing name"
+	default "myThingName"
+	help
+		Set the AWS Thing name created on IoT Console
+
+config AWS_SUBSCRIBE_TOPIC
+	string "MQTT subscribe topic"
+	default "myThingName/downlink"
+	help
+		MQTT topic the client should subscribe to.
+
+config AWS_PUBLISH_TOPIC
+	string "MQTT publish topic"
+	default "myThingName/data"
+	help
+		MQTT topic the client should publish to.
+
+choice AWS_TEST_SUITE
+	prompt "Device Advisor test suite"
+	default AWS_TEST_SUITE_NONE
+	help
+		Select the AWS Device Advisor test suite to run.
+config AWS_TEST_SUITE_NONE
+	bool "No test suite running"
+config AWS_TEST_SUITE_DQP
+	bool "Device Qualification Program (DQP)"
+	help
+		Make sure your region supports AWS Device Advisor for DQP
+config AWS_TEST_SUITE_RECV_QOS1
+	bool "Test suite for receiving QoS 1 messages"
+	help
+		For single test case "MQTT Client Puack QoS1"
```

Comment:
I'd still add an empty line in between choice options, currently, it's a bit hard to read (at least that was my first impression).